src/components/puck/registry/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "visual-editor-component-registry",
+  "version": "0.0.0",
+  "description": "Created by Yext",
```

Comment:
A component library registry for visual-editor components to work with the ShadCN CLI